TABTestKit/Classes/Biometrics/Biometrics.h

```diff
@@ -0,0 +1,43 @@
+//
+//  Biometrics.h
+//  TABTestKit
+//
+//  Created by Kane Cheshire on 31/10/2018.
+//  Copyright © 2018 The App Business. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface Biometrics : NSObject
+
+/*!
+ * Enroll biometrics on the simulator
+ */
++ (void)enrolled;
+
+/*!
+ * Unenroll biometrics on the simulator
+ */
++ (void)unenrolled;
+
+/*!
+ * Performs a successful biometric authentication on the simulator
+ * 
+ * Uses Touch ID ('fingerTouch') and FaceID ('pearl') to perform
```

Comment:
This is an implementation detail that doesn't need to be part of the header docs imo. These docs should just help explain what the function does, not how it does it

Reply:
Oh okay!

Reply:
Fixed!

Reply:
👌